purchase_ext_sst/models/purchase_order_line_hook_onchange_product_id.py

```diff
@@ -1,45 +1,55 @@
-# -*- coding: utf-8 -*-
 # Copyright 2018 Quartile Limited
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 
 from datetime import datetime
 
-from odoo import api, fields, models, SUPERUSER_ID, _
+from odoo import SUPERUSER_ID, api, models
 from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
 from odoo.addons.purchase.models.purchase import PurchaseOrderLine
 
+
 # Monkey Patching
 # Overwrite the original onchange_product_id
-# i.e. https://github.com/odoo/odoo/blob/11.0/addons/purchase/models/purchase.py#L781-L811
-@api.onchange('product_id')
+# i.e. https://github.com/odoo/odoo/blob/11.0
```

Comment:
Please use `# noqa` for this line, we do not want to split links into two lines.